backpropagation/network/neural_network.py

```diff
@@ -41,16 +41,34 @@ def __init__(
         self.layers_count = len(neurons_count_per_layer)
 
         self.weights = [
-            np.random.normal(size=(
+            self._create_weight_matrix(size=(
                 neurons_count_per_layer[layer + 1],
-                neurons_count_per_layer[layer] + 1
+                neurons_count_per_layer[layer]
```

Comment:
Why the difference?

Reply:
Now there is a separate function for creating the weight matrices which automatically adds the bias terms. Previously the random.normal function was used from the numpy library hence it had to be written explicitly that we need an extra column for bias terms.